deep_speech_2/cloud/pcloud_prepare_data.py

```diff
@@ -0,0 +1,61 @@
+"""
+This tool is used for preparing data for DeepSpeech2 trainning on paddle cloud.
+
+Steps:
+1. Read original manifest and get the local path of sound files.
+2. Tar all local sound files into one tar file.
+3. Modify original manifest to remove the local path information.
+
+Finally, we will get a tar file and a manifest with sound file name, duration
+and text.
+"""
+import json
+import os
+import tarfile
+import sys
+import argparse
+sys.path.append('../')
+from data_utils.utils import read_manifest
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument(
+    "--manifest_path",
+    default="../datasets/manifest.train",
+    type=str,
+    help="Manifest of target data. (default: %(default)s)")
+parser.add_argument(
+    "--out_tar_path",
+    default="./data/dev.tar",
+    type=str,
+    help="Output tar file path. (default: %(default)s)")
+parser.add_argument(
+    "--out_manifest_path",
+    default="./data/dev.mani",
```

Comment:
I renamed it to cloud.train.manifest. Use manifest as suffix for indicating the format of this file?